src/bvals/comms/bnd_info.hpp

```diff
@@ -56,6 +56,16 @@ struct BndInfo {
   SpatiallyMaskedIndexer6D idxer[3];
   forest::LogicalCoordinateTransformation lcoord_trans;
 
+  // Number of points contained in this boundary region
+  KOKKOS_FORCEINLINE_FUNCTION
+  int size() const {
+    int s = 0;
+    for (int n = 0; n < ntopological_elements; ++n) {
```

Comment:
Almost certainly doesn't matter, but should these be `size_t`s? (Maybe not if you send them?)

Reply:
oh, yeah, probably should return `size_t`. Switched.